### ISSUE TITLE
[Questions] All IPA releases gone missing?

### ISSUE THREAD

**Issue by iam-thedude** [arichornlover/uYouEnhanced#432]
### Have you read the FAQ?

- [X] Yes, I did read the FAQ

### Is there an existing issue/question for this?

- [X] I have searched the existing issues

### Do you think this is a bug?

✅ Yes, I believe this is a bug. I will open a new issue with the bug template

### My question

Hi!

So, why IPA releases gone missing?

**Comment by yodaluca23** [comment]
> I got DMCA’d by Google.

Not just uYouEnhanced btw @iam-thedude all tweaked YouTube apps on GitHub.

**Comment by Kylesare57** [comment]
It went down this morning, I downloaded an ipa like 2 hours before it went down though.

aHR0cHM6Ly9kcml2ZS5nb29nbGUuY29tL2ZpbGUvZC8xOWFUUERLbUdFZlhwd0MybXBfb2NnUHpfVFlfaWxja0Evdmlldz91c3A9ZHJpdmVzZGs=

**Comment by Kylesare57** [comment]
> ### Have you read the FAQ?
> * [x]  Yes, I did read the FAQ
> 
> ### Is there an existing issue/question for this?
> * [x]  I have searched the existing issues
> 
> ### Do you think this is a bug?
> ✅ Yes, I believe this is a bug. I will open a new issue with the bug template
> 
> ### My question
> Hi!
> 
> So, why IPA releases gone missing?
> 
> ### Additional context
> _No response_

It went down this morning, I downloaded an ipa like 2 hours before it went down though.

aHR0cHM6Ly9kcml2ZS5nb29nbGUuY29tL2ZpbGUvZC8xOWFUUERLbUdFZlhwd0MybXBfb2NnUHpfVFlfaWxja0Evdmlldz91c3A9ZHJpdmVzZGs=

**Comment by hi-im-karma** [comment]
So what now? Can we go on Telegram to get the ipa releases?

**Comment by Shaukariya** [comment]
> I got DMCA’d by Google. I had to do whatever I did to protect this repository.
> 
> 
> 
> read more about the DMCA Situation below.
> 
> https://www.reddit.com/r/jailbreak/comments/1b86tiz/balackburn_here_maintainer_of_ytliteplus_ive_just/

Dammit where do I get the new releases then? Wait is ur altstore repo gone too?

**Comment by Kylesare57** [comment]
> > I got DMCA’d by Google. I had to do whatever I did to protect this repository.
> > read more about the DMCA Situation below.
> > https://www.reddit.com/r/jailbreak/comments/1b86tiz/balackburn_here_maintainer_of_ytliteplus_ive_just/
> 
> Dammit where do I get the new releases then? Wait is ur altstore repo gone too?

refer to my previous post. You could say it’s based, base64 even

**Comment by Kylesare57** [comment]
> > I got DMCA’d by Google. I had to do whatever I did to protect this repository.
> > read more about the DMCA Situation below.
> > https://www.reddit.com/r/jailbreak/comments/1b86tiz/balackburn_here_maintainer_of_ytliteplus_ive_just/
> 
> Dammit where do I get the new releases then? Wait is ur altstore repo gone too?

im still able to download it from their alt store repo

**Comment by Shaukariya** [comment]
> uYouEnhanced

The uYou and uYouPlus repos are still downloadable to

**Comment by hi-im-karma** [comment]
Seems like they deliberately gave everyone only 1 day to comply with their requests. Knowing that they might miss the message. 
What a bunch of sneaky spineless lowlifes Google are...

**Comment by yodaluca23** [comment]
> Seems like they deliberately gave everyone only 1 day to comply with their requests. Knowing that they might miss the message. 
> What a bunch of sneaky spineless lowlifes Google are...

That's GitHub's DMCA takedown policy, has nothing to do with Google...

https://docs.github.com/en/site-policy/content-removal-policies/dmca-takedown-policy#d-what-if-i-inadvertently-missed-the-window-to-make-changes

**Comment by hi-im-karma** [comment]
> That's GitHub's DMCA takedown policy, has nothing to do with Google...

What does it matter whose policy it is.. It serves Google well.

**Comment by samuelawachie** [comment]
I just thank God that the tweaked app is at a very mature stage of development. This would’ve been catastrophic were it done a couple years ago. 
Everything I need is already in the app. 😇😎

**Comment by samuelawachie** [comment]
Lol @hi-im-karma going around to downvote all my comments coz I disagreed with him on an earlier thread. Get a life, dude.

**Comment by gototheskinny** [comment]
Is there any plans to move the project to other GitHub clone sites? Or is this tweak dead for good?

**Comment by gototheskinny** [comment]
> > Is there any plans to move the project to other GitHub clone sites? Or is this tweak dead for good?
> 
> 
> 
> I don’t know, I think the tweak is fine. All I can’t do is upload any IPAs to the public.

I hope Google scrape off this from the site. I don't know if it's the same with the ipas I hope not?

**Comment by KingGamingYT** [comment]
> They broke v16.xx.x YouTube versions today as of March 7th 2024. Is that a weird coincidence? shutting down older versions around the time we took down the IPAs.

Not a coincidence at all, they shut down v16.xx.x around two weeks ago for a day and then brought it back up, looks like they're now just beginning to take it down once again.

**Comment by KingGamingYT** [comment]
> > > They broke v16.xx.x YouTube versions today as of March 7th 2024. Is that a weird coincidence? shutting down older versions around the time we took down the IPAs.
> > 
> > 
> > Definitely a coincidence, they shut down v16.xx.x around two weeks ago for a day and then brought it back up, looks like they're now just beginning to take it down once again.
> 
> Also you can be able to see that the option “Fake YouTube Version” in the YouPiP Settings doesn’t work anymore today. So it definitely does confirm they’re gone.

This is true, but you can see it attempt to do so with the option enabled. It simply fails and loads the newer layout instead. I may be wrong in this but the _player_ itself from v16.xx.x still works, so if you don't press reload on the watch page you'll still have the fullscreen player without all the extra buttons at the bottom below the progress bar, only recommended videos.

**Comment by Kylesare57** [comment]
> > Is there any plans to move the project to other GitHub clone sites? Or is this tweak dead for good?
> 
> I don’t know, I think the tweak is fine. All I can’t do is upload any IPAs to the public.

Can you do what YouTube recanted does? Make a launcher that requires you to supply the ipa and inject into that

**Comment by KingGamingYT** [comment]
> Can you do what YouTube revanced does? Make a launcher that requires you to supply the ipa and inject into it? It doesn’t need to be signed, just available for us to sign and install easily

I don't think this is possible on iOS, at least when jailed.

**Comment by Kylesare57** [comment]
> > Can you do what YouTube revanced does? Make a launcher that requires you to supply the ipa and inject into it? It doesn’t need to be signed, just available for us to sign and install easily
> 
> I don't think this is possible on iOS, at least when jailed.

Not to install the app but I’m asking for one that modifies an ipa we supply instead of compiling it on a computer

**Comment by tony971** [comment]
Supplying my own ipa link to build myself, I'm getting the following error in github actions:

>The domain/default pair of (/Users/runner/work/uYouEnhanced/uYouEnhanced/main/Payload/YouTube.app/Info, CFBundleVersion) does not exist
>Error: Process completed with exit code 1.

Is there something I need to tweak about the ipa?

**Comment by ccarlos899** [comment]
Yeah ATP we should move to another platform, like telegram or something else

**Comment by Kylesare57** [comment]
> > They broke v16.xx.x YouTube versions today as of March 7th 2024. Is that a weird coincidence? shutting down older versions around the time we took down the IPAs.
> 
> Also another weird coincidence, the downloading videos/audio functionality broke aswell on YouTube Reborn. https://github.com/arichornlover/YouTube-Reborn-v5 / https://github.com/arichornlover/YouTubeRebornPlus Google DEFINITELY messed with the app after the removal of the ipa files.

weird I was having this issue at least since 19.08.2

**Comment by isibizi** [comment]
Hello, i have tried to join the telegram group, but i‘m kickicked out immediately after joined the group , why?

**Comment by KarmaBeast** [comment]
> > Hello, i have tried to join the telegram group, but i‘m kickicked out immediately after joined the group , why?
> 
> 
> 
> I never made a Telegram group and don’t even use that app. If there is a group related to the uYouEnhanced tweak then I cannot say it’s official.

It's being run by me and few other admins including bhackel. We make it clear that it's not an "official group", and that you, the developer, haven't joined it. We do not pretend to represent this repository or anything like that.

**Comment by Kylesare57** [comment]
> Hello, i have tried to join the telegram group, but i‘m kickicked out immediately after joined the group , why? 

We have a bot checker, when you join it asks a basic addition math problem to stay
